### PR TITLE
Conference report changes

### PR DESCRIPTION
**Pull request by sertansenturk**
Note that Github changes the file encoding from windows-1252 to UTF-8. @alexanderlerch I'm not sure if it breaks your setup.

### CODE REVIEW

File: ISMIR2021.tex [hunk 1]
```diff
@@ -653,28 +653,28 @@ \section{Conference Program}
...
+            Masterclasses are sessions where panelists discuss topics related to getting into and working in the industry, and they are intended to familiarize current students and recent graduates with interviews and processes they may encounter when looking for MIR-style roles in industry. These sessions were not tied to sponsorship, i.e., the panelists were approached independent of their companies’ participation in the conference.
              \begin{itemize}
                 \item Masterclass 1: CV
                     \begin{itemize}
                         \item   Time: 3.00-4.00 AM on November 12, 2021 (UTC)
-                        \item   Panelists: Christopher Bakes (ByteDance/TikTok,USA), Lamtharn �Hanoi� Hantrakul (ByteDance/TikTok, China)
+                        \item   Panelists: Christopher Bakes (ByteDance/TikTok,USA), Lamtharn “Hanoi” Hantrakul (ByteDance/TikTok, China)
```
Comment (sertansenturk) [inline ISMIR2021.tex:672]:
```suggestions
                        \item   Panelists: Christopher Bakes (ByteDance/TikTok, USA), Lamtharn “Hanoi” Hantrakul (ByteDance/TikTok, China)
```

Comment (sertansenturk) [reply ISMIR2021.tex:672]: ☝️ This should have created a commitable suggestion. Not sure why it didn't work. I'll check it later.

Comment (alexanderlerch) [reply ISMIR2021.tex:672]: Would probably be better to use `` '' for all quotation marks

**Comment by sertansenturk** [comment]
@alexanderlerch I have finished checking the relevant bits of the document, and there were only a few minor issues. I fixed them all except for making Fig 5 and Fig 6 smaller. I assume you will re-size all figures later.

Please feel free to merge the PR at your convenience. Thanks a lot!

**Review by alexanderlerch — APPROVED** [review]
Looks all good to me

**Pull request merged**